src/ExpressionEvaluator/Package/source.extension.vsixmanifest

```diff
@@ -4,25 +4,14 @@
     <Identity Id="21BAC26D-2935-4D0D-A282-AD647E2592B5" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Expression Evaluators</DisplayName>
     <Description xml:space="preserve">Roslyn Expression Evaluators</Description>
-    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\ExpressionEvaluators</ShortcutPath>
+    <PackageId>Microsoft.CodeAnalysis.ExpressionEvaluator</PackageId>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
-  <Installer>
-    <Actions>
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.ExpressionCompiler.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.ResultProvider.dll" />
-      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.ExpressionEvaluator.ExpressionCompiler.dll" />
```

Comment:
..\Core\Source\ExpressionCompiler\ExpressionCompiler.csproj

---
In reply to: [91194641](https://github.com/dotnet/roslyn/pull/15710#discussion_r91194641) [](ancestors = 91194641)